SDWebImage/FLAnimatedImage/FLAnimatedImageView+WebCache.m

```diff
@@ -120,9 +127,13 @@ - (void)sd_setImageWithURL:(nullable NSURL *)url
                              options:options
                         operationKey:nil
                        setImageBlock:^(UIImage *image, NSData *imageData) {
+                           onExit {
```

Comment:
Emm, we can't use `cleanup` attribute in here.

Reply:
This PR is closed. Because it's not related to that issue due to my fault.